e2e/actions/actions.go

```diff
@@ -302,7 +302,7 @@ func (c actionTests) actionShell(t *testing.T) {
 			argv: []string{c.env.ImagePath},
 			consoleOps: []e2e.SingularityConsoleOp{
 				e2e.ConsoleSendLine("hostname"),
-				e2e.ConsoleExpect(hostname),
+				e2e.ConsoleExpect("hostname"),
```

Comment:
This seems to be expecting the `hostname` command to be echoed back, but it is no longer checking that the command returns the hostname that we expect.